### PR TITLE
Proposed update to the Accordion component to allow for a "expand/collapse all" toggle

### PR DESCRIPTION
**Pull request by bingles**
This update modifies the "Accordions" component by adding a "expand/collapse all" toggle option for the multi-select variant. It breaks the component up into two distinct sections: Behaviors and Styles. Would love to discuss further in depth with the group.

### CODE REVIEW

File: docs/components/accordions.md [hunk 1]
```diff
@@ -11,16 +11,119 @@ Users need to see only relevant content.
...
+      collapseThree.collapse('hide');
+   });
+
+  $('#expand-all').click(function(){
+      collapseOne.collapse('show');
+      collapseTwo.collapse('show');
+      collapseThree.collapse('show');
+  });
+
+}, 100);
```
Comment (bmathews) [inline docs/components/accordions.md]: Awesome! I think this particular bit will throw errors if jQuery takes longer than 100ms to load, yeah? Maybe add it to the `fornt/scripts/main.js` file (which is just JS to facilitate the demos/examples) with a hook like `#exampleToggler` on your `.panel-toggler` div? Something like: http://github.com/USPTO/designpatterns/blob/gh-pages/front/scripts/main.js#L200

Comment (bingles) [inline docs/components/accordions.md]: Thanks Brian, thats a good idea! Had some help on this and was wondering where the best place for it to live would be.

**Comment by AaronUXD** [comment]
Two cents –

An “accordion” pattern normally refers to only the first option, when any section is selected, the others collapse. It’s meant specifically for instances with limited vertical space.

Wouldn’t the other two options be covered by the “expandable rows” option under the “Table” component?

Possible compromise, call the component something else, label the first option “accordion”, label the other two something else?

------------------------------------
aaron.pepe@uspto.gov
(571) 272-7185 | MDW 4D30

**Comment by scottmerker** [comment]
Generally those would be called collapsible groups or collapsible panels if you can expand more than one at the same time.   If we are talking about the same thing...



—

**Pull request merged**